### PR TITLE
MOSIP-12748 Fixed CACert sync issue

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: registration/registration-services/src/main/java/io/mosip/registration/mdm/service/impl/MosipDeviceSpecificationHelper.java [hunk 1]
```diff
@@ -66,7 +66,7 @@ public String getPayLoad(String data) throws RegBaseCheckedException {
 
 	public MdmDeviceInfo getDeviceInfoDecoded(String deviceInfo) {
 		try {
-			validateJWTResponse(deviceInfo);
+			validateJWTResponse(deviceInfo, "DEVICE");
```
Comment (gsasikumar) [inline registration/registration-services/src/main/java/io/mosip/registration/mdm/service/impl/MosipDeviceSpecificationHelper.java]: please define a constant and pick from there. or can we not pick this from properties?

File: registration/registration-services/src/main/java/io/mosip/registration/mdm/spec_0_9_2/service/impl/MosipDeviceSpecification_092_ProviderImpl.java [hunk 2]
```diff
@@ -322,7 +322,7 @@ private MdmBioDevice getBioDevice(MdmDeviceInfo deviceInfo)
 	}
 
 	private DigitalId getDigitalId(String digitalId) throws IOException, RegBaseCheckedException, DeviceException {
-		mosipDeviceSpecificationHelper.validateJWTResponse(digitalId);
+		mosipDeviceSpecificationHelper.validateJWTResponse(digitalId, "FTM");
```
Comment (gsasikumar) [inline registration/registration-services/src/main/java/io/mosip/registration/mdm/spec_0_9_2/service/impl/MosipDeviceSpecification_092_ProviderImpl.java]: Please define the constant

**Review by gsasikumar — CHANGES_REQUESTED** [review]

File: registration/registration-services/src/main/java/io/mosip/registration/mdm/service/impl/MosipDeviceSpecificationHelper.java (hunk 1 above)
Comment (ghost) [reply registration/registration-services/src/main/java/io/mosip/registration/mdm/service/impl/MosipDeviceSpecificationHelper.java]: changed to use properties

File: registration/registration-services/src/main/java/io/mosip/registration/mdm/spec_0_9_2/service/impl/MosipDeviceSpecification_092_ProviderImpl.java (hunk 2 above)
Comment (ghost) [reply registration/registration-services/src/main/java/io/mosip/registration/mdm/spec_0_9_2/service/impl/MosipDeviceSpecification_092_ProviderImpl.java]: changed to use properties

**Review by gsasikumar — APPROVED** [review]

**Pull request merged**